### PR TITLE
[CELEBORN-826][DOC] Add storage document

### PR DESCRIPTION
**Pull request by waitinfuture**
### What changes were proposed in this pull request?
As title.

### Why are the changes needed?
As title.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
No.

### CODE REVIEW

**Comment by waitinfuture** [comment]
cc @pan3793 @cfmcgrady @AngersZhuuuu @RexXiong

File: client/src/test/scala/org/apache/celeborn/client/WorkerStatusTrackerSuite.scala [hunk 1]
```diff
@@ -30,6 +30,22 @@ import org.apache.celeborn.common.protocol.message.StatusCode
...
+      if (v > 0) {
+        println("haha")
+        return
+      }
+      println("aaa")
+    }
+    )
+  }
+
+  test("test") {
```
Comment (cfmcgrady) [inline client/src/test/scala/org/apache/celeborn/client/WorkerStatusTrackerSuite.scala:45]: Oops...

**Review by cfmcgrady — APPROVED** [review]
LGTM

**Comment by waitinfuture** [comment]
Thanks, merged to main.

**Pull request closed without merging**